src/parser/paladin/holy/modules/azeritetraits/GlimmerOfLight.js

```diff
@@ -157,7 +157,7 @@ class GlimmerOfLight extends Analyzer {
           </>
         )}
         tooltip={(
-          <Trans>
+          <>
```

Comment:
I am not sure why this is.  The <Trans> tag works fine when I build locally but when it is deployed it does not complete the string formatting.